Files/Models/Settings/BundlesSettingsModel.cs

```diff
@@ -11,7 +12,7 @@ public class BundlesSettingsModel : BaseJsonSettingsModel, IBundlesSettings
 
         public BundlesSettingsModel()
             : base(System.IO.Path.Combine(ApplicationData.Current.LocalFolder.Path, Constants.LocalSettings.SettingsFolderName, Constants.LocalSettings.BundlesSettingsFileName),
-                  isCachingEnabled: true)
+                  isCachingEnabled: false)
```

Comment:
Set `isCachingEnabled` to prevent from breaking Bundles serialization

Reply:
Re-added the caching feature in https://github.com/d2dyno1/Files/commit/fb0fa9da31a2f6942150c4d09180dc0a1afcdeb2